UI/org.eclipse.birt.report.designer.ui.rcp/BIRT.product

```diff
@@ -1,33 +1,64 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?pde version="3.5"?>
 
-<product name="%ApplicationName" id="org.eclipse.birt.report.designer.ui.rcp.BIRT" version="4.17.0.qualifier" application="org.eclipse.birt.report.designer.ui.rcp.DesignerApplication" useFeatures="false" includeLaunchers="true">
+<product name="BIRT Report Designer All-In-One" uid="org.eclipse.birt.report.designer.all" id="org.eclipse.birt.branding.birt_all_in_one" application="org.eclipse.ui.ide.workbench" version="4.17.0.qualifier" type="features" includeLaunchers="true" autoIncludeRequirements="true">

```

Comment:
I don't think we even ship this product.   This is the only one we ship:

![image](https://github.com/eclipse-birt/birt/assets/208716/bead9bc9-1951-4a4a-9dd8-496684ae216f)

Let's not change stuff that is effectively garbage as far as I can tell.

Reply:
Sorry my bad, I used github to make the quick change.